lib/workers/repository/update/pr/index.ts

```diff
@@ -93,14 +93,20 @@ export async function ensurePr(
   prConfig: BranchConfig
 ): Promise<EnsurePrResult> {
   const getBranchStatus = memoize(() =>
-    resolveBranchStatus(branchName, ignoreTests)
+    resolveBranchStatus(branchName, !!internalChecksAsSuccess, ignoreTests)
```

Comment:
looks a little bit strange to use variables before declaration. 🤔

can we move the the variables before this line?

Reply:
Will change it in a refactor PR